client/src/components/AdminCard/style.css

```diff
@@ -0,0 +1,101 @@
+.card-container {
+    width: 80%;
+    margin: 50px auto;
+}
+
+.card-container ul li {
+    display: grid;
+    grid-template-columns: 15% 21% repeat(4, 12.6%) [last-col] 13.5%;
+    box-shadow: var(--card-shadow);
+    margin-bottom: 30px;
+    padding: 20px 0;
+}
+
+.card-container ul li > div:last-of-type {
+    grid-column: last-col / span 1;
+}
+
+.card-container ul li > div {
+    border-left: 1px dashed #C8C7C7;
+    padding: 0 20px;
+}
+
+.card-container ul li > div:first-of-type {
+    border-left: none;
+}
+
+.card-container ul li .name-img-card {
+    display: flex;
+    justify-content: space-between;
+}
+
+.card-container ul li .name-img-card > div {
+    width: 40%;
+}
+
+.card-container ul li .img-card {
+    align-self: center;
+}
+
+.card-container ul li .name-img-card img {
+    width: 50px;
+}
+
+.card-container ul li h3 {
+    color: var(--main-color);
+    margin-bottom: 11px;
+}
+
+.card-btn {
+    display: block;
+    width: 90px;
+    border-color: var(--border-white);
+    color: #414D55;
+    border-radius: 5px;
+    margin: 0 auto;
+}
+
+.card-btn:hover {
+    box-shadow: 0px 0px 8px 1px #f3f2f3;
```

Comment:
I think it will be better if you used the existing shadow variable var(--box-shadow);

Reply:
I used the existing variables, but here the shadow effects are not the same in the existing shadow effects so I need to add another one.

Reply:
Yes I know but from my opinion there is no actual diferences between the two color on the screen, so i said that.